canonical-websites/www.ubuntu.com
# Add new MAAS takeover

## Done

Add new MAAS takeover

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/))
- Check the homepage takeover in s, m and l

## Issue / Card

Card: https://trello.com/c/mI6jelii
Doc: https://docs.google.com/document/d/1IbyGV-s-LVFzGZCeE7nSo4ynAfvqiZLmqfNE4swddLQ/edit
Designs: https://www.dropbox.com/home/00_web_team/_ubuntu.com/takeovers/MAAS%2005.17/Exportables
